layouts/partials/fragments/copyright.html

```diff
@@ -48,11 +48,11 @@
           </div>
         </div>
       {{ end }}
-      {{ if $menus.copyright_footer }}
+      {{ if .Site.Menus.legal_footer }}
```

Comment:
We are not quite consistent with the naming. The fragment is copyright. The translation key is legalFooter and the menu is legal_footer. Probably should be all moved to copyright or something more abstract.

Reply:
Fixed in 02991898802d2faa7c8be867619a655643a2edb9